webroot/css/cake.css

```diff
@@ -0,0 +1,562 @@
+.disabled a,
+a.disabled {
```

Comment:
I agree. Just because `cake.css` and `base.css` are referenced in the layouts https://github.com/cakephp/docs_search/blob/master/src/Template/Layout/default.ctp#L29-L30.

I can remove the css files if you prefer?